google/earthenterprise
# Update to scons 4.x

Updates our code to scons 4.x and updates our scons files to work with python 3.

To test:
1. Build and install
2. Verify that these things complete successfully
3. Verify with the tutorial data
